### DISCUSSION TITLE
Feature Request: Add Minimum Limit for MultiSelect

### DISCUSSION THREAD

**Discussion by nakabor** [charmbracelet/huh#550] [Ideas]
### Description
Currently, the `MultiSelect` component allows setting a maximum limit for selections using the `Limit()` method. However, there's no way to set a minimum number of required selections. Adding a minimum limit would enhance form validation capabilities.

### Proposed Solution
Introduce a new method, perhaps called `MinLimit(n int)`, to set the minimum number of selections required for a `MultiSelect` field.

P.S. `AtLeast(n int)` might be great as well.

### Example Usage
```go
huh.NewMultiSelect[string]().
    Title("Select at least 2 but no more than 4 toppings").
    Options(
        huh.NewOption("Cheese", "cheese"),
        huh.NewOption("Tomatoes", "tomatoes"),
        huh.NewOption("Onions", "onions"),
        huh.NewOption("Lettuce", "lettuce"),
        huh.NewOption("Pickles", "pickles"),
    ).
    MinLimit(2).  // New method to set minimum selections
    Limit(4).     // Existing method for maximum selections
    Value(&toppings)

**Comment by bashbunni** [comment]
Hey @nakabor, thank you so much for wanting to make this project better. I've converted this to a discussion so we can talk through it and keep our backlog clear. If it does end up being something we want to add, we'll promote it back to an issue when we are ready to work on it.

If we commit to this feature, that's a feature we will need to maintain for the lifespan of this project. So, since "Yes is forever", we are carefully considering new features.

I do think this feature would make sense, though naming-wise I would probably simplify to something like `Minimum`

**Comment by yutasb** [comment]
Hello!

I wanted this feature, so I went ahead and implemented it.
Would anyone be willing to review it?

https://github.com/charmbracelet/huh/pull/336
